lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -358,7 +384,16 @@ GLOW_ONNXIFI_LIBRARY_FUNCTION_WRAPPER(onnxInitGraph)(
     return ONNXIFI_STATUS_INVALID_BACKEND;
   }
 
-  auto *glowGraph = manager.createGraph(glowBackend);
+  glow::onnxifi::OnnxifiQuantizationStep quantizationStep;
+  if (getenv("GLOW_DUMP_PROFILE")) {
```

Comment:
that might be an effort not worth benefits, but `auxPropertiesList` could be a candidate to supply action instead of env vars.

Reply:
Yeah I was thinking about how we can more closely integrate this into c2 side of onnxifi in the future and I think that would be one way to go